src/widgets/courses/ui/courses.tsx

```diff
@@ -1,14 +1,18 @@
+import classNames from 'classnames/bind';
+import { path } from '../constants';
 import { COURSE_STALE_AFTER_DAYS } from '@/app/const';
 import { type Course, CourseCard } from '@/entities/course';
 import { getActualData } from '@/shared/helpers/getActualData';
 import { isCourse } from '@/shared/helpers/is-course';
 import { useDataByName } from '@/shared/hooks/use-data-by-name';
 import { WidgetTitle } from '@/shared/ui/widget-title';
 
-import './courses.scss';
+import styles from './courses.module.scss';
+
+const cx = classNames.bind(styles);
 
 export const Courses = () => {
-  const { data: courses, loading, error } = useDataByName('courses');
+  const { data: courses, loading, error } = useDataByName(path);
```

Comment:
I'd just use a string, since the `path` variable is currently only used in one place

Reply:
also proposed to return to the string, but after discussing with @Quiddlee , we decided to put in the constant
https://github.com/rolling-scopes/site/pull/545#discussion_r1775899673

Reply:
we'll delete this hook anyway as soon as we start fetch the courses from external cdn